### ISSUE TITLE
Storing the Database on an external drive

### ISSUE THREAD

**Issue by laudmusic** [PostgresApp/PostgresApp#742]
Is there a way to set PostgresApp to store the database on an external drive versus my internal boot drive? I have a space issue with a very large DB I need to create.
Thanks!
M

**Comment by jakob** [comment]
Yes. When you create a new server, you can select the data directory.

You can also try copying your existing data directory over, but that's something we don't test so proceed at your own risk.
